lib/apps/submit.py

```diff
@@ -25,7 +25,7 @@ def get(self):
         user = users.get_current_user()
 
         if not user:
-            warning = "WARNING, you are currently NOT signed in. The announcement must be approved by a registered user before the announcement will be displayed!"
+            warning = "Your email account is not registered for access to this application. If you are a [school name] staff member, you can register for access using the instructions provided by your school.  Contact your computer contact teacher if you need more information."
```

Comment:
replace the [school name] with the actual school name, either using %s of .format
You can get the school name by the
schoolQueryInfo and schoolInfo variables that are later declared.
Just make sure to check that school is not None